enterprise/wsproxy/wsproxy.go

```diff
@@ -235,9 +235,11 @@ func New(ctx context.Context, opts *Options) (*Server, error) {
 	r.Get("/healthz-report", s.healthReport)
 	r.NotFound(func(rw http.ResponseWriter, r *http.Request) {
 		site.RenderStaticErrorPage(rw, r, site.ErrorPageData{
-			Status:       404,
-			Title:        "Route Not Found",
-			Description:  "The route you requested does not exist on this workspace proxy. Maybe you intended to make this request to the primary dashboard? Click below to be redirected to the primary site.",
+			Title:      "Workspace Proxy",
```

Comment:
Thoughts on this being "Redirect to Dashboard" or "Head to the Dashboard" instead? Just minor for UX.

Reply:
On the button you mean?

Reply:
Well, the title doesn't really add to the content at all, so it's just redundant at the moment.

I do think the button should change too though, and say "Back to Dashboard" instead of "Back to Site"

Reply:
Ah. I can make the title that, we can change the button later if it's important.